src/cpuid/src/cpu_leaf.rs

```diff
@@ -296,13 +296,19 @@ pub mod leaf_0x80000001 {
     pub const LEAF_NUM: u32 = 0x8000_0001;
 
     pub mod ecx {
-        pub const TOPOEXT_INDEX: u32 = 22;
+        pub const SSE4A: u32 = 6; // EXTRQ, INSERTQ, MOVNTSS, and MOVNTSD instruction support
+        pub const MISALIGN_SSE: u32 = 7; // Misaligned SEE mode
         pub const PREFETCH_BITINDEX: u32 = 8; // 3DNow! PREFETCH/PREFETCHW instructions
+        pub const TOPOEXT_INDEX: u32 = 22; // topology extensions support
         pub const LZCNT_BITINDEX: u32 = 5; // advanced bit manipulation
```

Comment:
Fixes the order of bits (in ascending order).
```suggestion
        pub const LZCNT_BITINDEX: u32 = 5; // advanced bit manipulation
        pub const SSE4A: u32 = 6; // EXTRQ, INSERTQ, MOVNTSS, and MOVNTSD instruction support
        pub const MISALIGN_SSE: u32 = 7; // Misaligned SEE mode
        pub const PREFETCH_BITINDEX: u32 = 8; // 3DNow! PREFETCH/PREFETCHW instructions
        pub const TOPOEXT_INDEX: u32 = 22; // topology extensions support
```

Reply:
done